Lib/pathlib.py

```diff
@@ -1245,34 +1497,26 @@ def check_eloop(e):
                 check_eloop(e)
         return p
 
-    def owner(self):
-        """
-        Return the login name of the file owner.
-        """
-        try:
-            import pwd
+    if pwd:
+        def owner(self):
+            """
+            Return the login name of the file owner.
+            """
             return pwd.getpwuid(self.stat().st_uid).pw_name
-        except ImportError:
-            raise UnsupportedOperation("Path.owner() is unsupported on this system")
```

Comment:
This will change error messages from ``UnsupportedOperation("Path.owner() is unsupported on this system")`` to ``UnsupportedOperation("Path.owner()")`` -- is the information that the failure is system dependent useful here // will it be missed?

Reply:
It might be missed, yes. I could define a method like this and use it to raise exceptions - does that seem sensible to you?

```python
    def _unsupported(self, method):
        msg = f"{type(self).__name__}.{method} is unsupported"
        if isinstance(self, Path):
            msg += " on this system"
        raise UnsupportedOperation(msg)
```

Reply:
Yep, seems a good compromise.